thunder/core/trace.py

```diff
@@ -572,3 +572,16 @@ def _set_execution_file(path: str) -> None:
 
 def _get_execution_file() -> None | str:
     return _execution_file.get()
+
+
+#
+# Container for the two/three types of traces, plus extra tracked data
+#
+
+
+class JitResults:
```

Comment:
I wonder if there could be a better name for it, but I'm not entirely sure what it would be. I tend to think of the JIT result as the translated function, so this is more something like the first stage result or the initial tracing result or somesuch.

Reply:
Come to think of it, `TraceResults` seems like a better name.